src/locale/locales/de/messages.po

```diff
@@ -830,88 +813,85 @@ msgstr "Feed offline"
 msgid "Feed Preferences"
 msgstr "Feed-Einstellungen"
 
-#: src/view/shell/desktop/RightNav.tsx:65
-#: src/view/shell/Drawer.tsx:311
+#: src/view/shell/desktop/RightNav.tsx:65 src/view/shell/Drawer.tsx:311
 msgid "Feedback"
-msgstr "Rückmeldung"
+msgstr "Feedback"
 
-#: src/view/screens/Feeds.tsx:475
-#: src/view/screens/Profile.tsx:165
+#: src/view/screens/Feeds.tsx:475 src/view/screens/Profile.tsx:165
 #: src/view/shell/bottom-bar/BottomBar.tsx:181
-#: src/view/shell/desktop/LeftNav.tsx:340
-#: src/view/shell/Drawer.tsx:474
+#: src/view/shell/desktop/LeftNav.tsx:340 src/view/shell/Drawer.tsx:474
 #: src/view/shell/Drawer.tsx:475
 msgid "Feeds"
 msgstr "Feeds"
 
 #: src/view/com/auth/onboarding/RecommendedFeeds.tsx:57
 msgid "Feeds are created by users to curate content. Choose some feeds that you find interesting."
-msgstr "Feeds werden von Nutzern erstellt, um Inhalte zu kuratieren. Wählen Sie einige Feeds aus, die Sie interessant finden."
+msgstr "Feeds werden von Nutzern erstellt, um Inhalte zu kuratieren. Wähle einige Feeds aus, die du interessant findest."
 
 #: src/view/screens/SavedFeeds.tsx:156
 msgid "Feeds are custom algorithms that users build with a little coding expertise. <0/> for more information."
-msgstr "Feeds sind benutzerdefinierte Algorithmen, die von den Nutzern mit ein wenig Programmierkenntnissen erstellt werden. <0/> für weitere Informationen."
+msgstr "Feeds sind benutzerdefinierte Algorithmen, die Nutzer mit ein wenig Programmierkenntnisse erstellen. <0/> für mehr Informationen."
 
 #: src/view/screens/Search/Search.tsx:414
 msgid "Find users on Bluesky"
-msgstr "Benutzer auf Bluesky finden"
+msgstr "Nutzer auf Bluesky finden"
 
 #: src/view/screens/Search/Search.tsx:412
 msgid "Find users with the search tool on the right"
-msgstr "Benutzer mit der Suchfunktion auf der rechten Seite finden"
+msgstr "Finde Nutzer mit der Suchfunktion auf der rechten Seite"
 
 #: src/view/com/auth/onboarding/RecommendedFollowsItem.tsx:150
 msgid "Finding similar accounts..."
 msgstr "Suche nach ähnlichen Konten..."
 
 #: src/view/screens/PreferencesHomeFeed.tsx:102
 msgid "Fine-tune the content you see on your home screen."
-msgstr "Optimieren Sie die Inhalte, die Sie auf Ihrem Startbildschirm sehen."
+msgstr ""
 
 #: src/view/screens/PreferencesThreads.tsx:60
 msgid "Fine-tune the discussion threads."
-msgstr "Feinabstimmung der Diskussionsstränge."
+msgstr ""
 
 #: src/view/com/profile/ProfileHeader.tsx:538
 msgid "Follow"
-msgstr "Folgen Sie"
+msgstr "Folgen"
 
 #: src/view/com/auth/onboarding/RecommendedFollows.tsx:64
 msgid "Follow some users to get started. We can recommend you more users based on who you find interesting."
-msgstr "Folgen Sie einigen Nutzern, um loszulegen. Wir können Ihnen weitere Nutzer empfehlen, je nachdem, wen Sie interessant finden."
+msgstr "Folge einigen Nutzern, um loszulegen. Wir können dir weitere Nutzer empfehlen, je nachdem, wen du interessant findest."
```

Comment:
Gender-neutral: "Nutzern"/"Nutzer" -> `"Personen"`

Reply:
`"Profilen"`

Reply:
While `Personen` provides a gender neutral option, I think `Profil(en)` is way better here, since individual user accounts aren't necessarily actual people, but might be companies, brands, etc. too.